frontend/src/components/ApplicationForm.jsx

```diff
@@ -1,15 +1,88 @@
 import Form from "react-bootstrap/Form"
+import { useState } from "react";
+import { Col, Row} from "react-bootstrap";
+import 'bootstrap/dist/css/bootstrap.min.css';
 
 const ApplicationForm = (props) => {
   // initialize state below this line
+  const [formPersonalInfo, setPersonalInfo] = useState({});
 
   // create any event handler functions below this line
 
+  const changePersonalInfo = (fieldName, value) => {
+    setPersonalInfo({...formPersonalInfo, [fieldName]: value}); 
+  }
+
   // all html related material below here
   return (
     <Form>
       <p>this is the form</p>
       {/* Personal Information Section below this line*/}
+      <Row>
+        <Col>
+          <Form.Group>
+            <Form.Label>Name (First, Last)</Form.Label>
+            <Form.Control required type ="text"
+            onChange={(e) => {changePersonalInfo("name", e.target.value)}}>
+            </Form.Control>
```

Comment:
Change all of the <Form.Control> elements to be...
![image](https://user-images.githubusercontent.com/69172929/168441176-63cd95b4-7f61-459a-8a0a-523a4a6f8059.png)

Reply:
align the curly braces to the start of the onChange

Reply:
do the same thing for the rest of the onChange handlers